avm/res/sql/server/encryption-protector/main.bicep

```
@@ -8,10 +8,10 @@ param sqlServerName string
 @description('Required. The name of the server key.')
 param serverKeyName string
 
-@description('Optional. Key auto rotation opt-in.')
-param autoRotationEnabled bool = false
+@description('Optional. Key auto rotation opt-in flag. Either true or false.')
+param autoRotationEnabled bool = true
 
-@description('Optional. The encryption protector type.')
+@description('Optional. The encryption protector type like \'ServiceManaged\', \'AzureKeyVault\'.')
```

```suggestion
@description('Optional. The encryption protector type.')
```
The allowed values are anyways provided to the user when calling the parameter